src/engine/engine.cpp

```diff
@@ -37,47 +40,55 @@ namespace CityFlow {
 
 
     bool Engine::loadConfig(const std::string &configFile) {
-        Json::Value root;
-
-        std::ifstream jsonfile(configFile, std::ifstream::binary);
-        if (!jsonfile.is_open()) {
+        FILE* fp = fopen(configFile.c_str(), "r");
+        if (!fp) {
             std::cerr << "cannot open config file!" << std::endl;
             return false;
         }
-        jsonfile >> root;
-        jsonfile.close();
-
-        interval = root["interval"].asDouble();
-        warnings = false;
-        rlTrafficLight = root["rlTrafficLight"].asBool();
-
-        if (root.isMember("laneChange"))
-            laneChange = root["laneChange"].asBool();
-        else
-            laneChange = false;
-
-        int seed = root["seed"].asInt();
-        rnd.seed(seed);
+        char readBuffer[65536];
```

Comment:
I am not sure what will happen if the file length is larger than 65536

Reply:
this is just a buffer for stream reader, so it should not be a concern? @soodoshll please verify

Reply:
testcase is much larger than 64k and it seems no problem here

Reply:
Since we don't know the effect of the buffer length, please change the buffer size to a const variable which is defined as a hyperparameter.

Reply:
agree @soodoshll